docs/includes/widgets/common.md

```diff
@@ -0,0 +1,29 @@
+# Widget_Common - `common.php`
+
+## Technical Description:
+
+This class is responsible for registering the controls under the `Advanced` tab of each widget in Elementor.
+
+
+## Features:
+
+### Mask:
+We've added a mask option to any type of widget in Elementor, using the CSS `mask` property.
+
+#### Knowledge Base:
+- [Mask Option on Elementor.com](https://elementor.com/help/mask-option/)
```

Comment:
Please add links to Elementor docs & videos of this feature.

Reply:
Good idea! :)